proof/invariant-abstract/AARCH64/ArchFPU_AI.thy

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: GPL-2.0-only
  *)
 
+(* Show that the TCB that owns the current FPU is in the current domain. *)
```

Comment:
um, that's the comment you want for the ArchFPU_AI theory? you sure nothing else is going to go in here later?

Reply:
I'm a bit lost. This comment is above `ArchFPU_AI` ... are you saying you can now put `ArchVCPUDomain_AI`'s contents into `ArchFPU_AI` and simplify the hierarchy? Worth a try, would simplify the hierarchy